python/deadlinesubmissionsettings/app.py

```diff
@@ -287,9 +291,9 @@ def _apply_settings(self):
 
         pools = [i for i in settings.get("pools", "").split(";") if i != ""]
         if not pools:
-            pools = ["none", "none"]
+            pools = ["none", "-"]
         elif len(pools) == 1:
-            pools.append("none")
+            pools.append("-")
 
```

Comment:
Should it `assert len(pools) == 2` or something to ensure it's not bugged with a third or more attribute? (Say someone broke it?)